css/uqcs.css

```diff
@@ -89,7 +89,7 @@ body {
 
 .navbar-fixed
 {
-    margin-bottom: 64px; /* Default height of Materialize Navbar */
+    margin-bottom: 56px; /* Default height of Materialize Navbar */
```

Comment:
What's the motivation behind this change? Before/after pics?

Reply:
There was a gap here that only showed up on mobile/smallest view size.
<img width="404" alt="screen shot 2018-03-12 at 10 32 04 am" src="https://user-images.githubusercontent.com/9084563/37289749-cc4af1c0-25e0-11e8-863c-df391232ad44.png">
<img width="402" alt="screen shot 2018-03-12 at 10 32 36 am" src="https://user-images.githubusercontent.com/9084563/37289753-cdfcc8a4-25e0-11e8-9ef4-7e18d9337637.png">

Reply:
Ah yep, good catch!